src/components/Button/Button.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Button.css'
+
+function Button (props) {
+    return (
+       <section className="outer-coll">
+           <h1>{props.fruitTitle}</h1>
```

Comment:
Ik zou de Button component echt alleen een Button geven, Hier is het alsnog dat je alles doorgeeft maar niet gebruik zal maken van de Button.